src/types/enums.ts

```diff
@@ -1,33 +1,63 @@
+/**
+ * Enum representing different supported blockchain networks.
+ */
 export enum Networks {
+    /** The Ethereum mainnet. */
     Ethereum = 'ethereum',
+    /** The Holesky testnet. */
     Holesky = 'holesky',
+    /** The Hardhat local development network. */
     /** @ignore */
     Hardhat = 'hardhat',
 }
+
+/**
+ * Enum for types of actions related to vault operations.
+ */
 export enum VaultActionType {
-    Redeemed = 'Redeemed',
-    Migrated = 'Migrated',
-    Deposited = 'Deposited',
+    /** Vault creation. */
     VaultCreated = 'VaultCreated',
+    /** Deposit into the vault. */
+    Deposited = 'Deposited',
+    /** Redemption of assets from the vault. */
+    Redeemed = 'Redeemed',
+    /** Claiming exited assets. */
+    ExitedAssetsClaimed = 'ExitedAssetsClaimed',
+    /** Minting osToken. */
     OsTokenMinted = 'OsTokenMinted',
+    /** Burning osToken. */
     OsTokenBurned = 'OsTokenBurned',
+    /** Redemption of osToken. */
     OsTokenRedeemed = 'OsTokenRedeemed',
+    /** Liquidation of osToken. */
     OsTokenLiquidated = 'OsTokenLiquidated',
-    ExitedAssetsClaimed = 'ExitedAssetsClaimed',
+    /** Migration from StakeWise v2. */
+    Migrated = 'Migrated',
 }
 
 /**
- * The position health will track the value of osETH minted by stakers relative to the value of their ETH stake in the Vault.
- * - `Healthy`: value of minted osETH does not exceed 90% of the staked ETH value a staker has in the Vault.
- * - `Moderate`: value of minted osETH exceeds 90% of the staked ETH value a staker has in the Vault but remains below 91%.
- * - `Risky`: value of minted osETH exceeds 91% of the staked ETH value a staker has in the Vault but remains below 92%.
- * - `Unhealthy`: value of minted osETH exceeds 92% of the staked ETH value a staker has in the Vault.
+ * Position health tracks the value of osETH minted by stakers relative to the value of their ETH stake in the vault.
+ * Healthy positions have minted osETH that is well-collateralized by staked ETH. As the proportion of minted osETH
+ * increases relative to staked ETH, position health deteriorates.
+ *
+ * Factors affecting position health include yield discrepancies (APY) between the vault and osETH, which can result
+ * from:
+ * - Differences in fee structures.
+ * - Variations in attestation performance.
+ * - The ratio of unbounded ETH to the vault's total value locked (TVL).
+ * - Delays in validator activation on the Beacon Chain.
+ * - Losses due to maximal extractable value (MEV) strategies.
  *
- * Please reffer to the official [documentation](https://docs.stakewise.io/protocol-overview-in-depth/oseth#position-health) for more details.
```

Comment:
I think it still makes sense to preserve the link here, just after explaining about position risk, like

```
Risky positions may enter redemption processes, while positions 
deemed unhealthy are subject to liquidation. 
See [stakewise documentation](https://docs.stakewise.io/protocol-overview-in-depth/oseth#position-health) for more details
```